packages/ast-spec/src/base/SourceLocation.ts

```diff
@@ -1,12 +1,16 @@
-import type { LineAndColumnData } from './LineAndColumnData';
+import type { Position } from './Position';
 
 export interface SourceLocation {
+  /**
+   * ESLint does not use the source property.
+   */
+  source?: string | null;
```

Comment:
(sorry I missed this earlier)

We don't emit this. Looking at other estree parsers (esprima, espree, babel, acorn) - none of them emit it either.
So we shouldn't add this.